frontend/plugins/resources/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@internal/plugin-resources",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core": "^0.4.0",
+    "@backstage/theme": "^0.2.2",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.45",
```

Comment:
I think it should be `"^4.0.0-alpha.57", with the caret `^` and latest alpha.

Reply:
I have put latest version of alpha. But didn't put caret since there isn't in services plugin. But if you insist, i will :)